src-plugins/reformat/shapeBasedInterpolation.cpp

```diff
@@ -460,15 +456,12 @@ shapeBasedInterpolation::~shapeBasedInterpolation()
     d = NULL;
 }
 
-void shapeBasedInterpolation::setInput(MaskType::Pointer input0, MaskType::Pointer input1, MaskType::Pointer input2, MaskType::Pointer input3)
+void shapeBasedInterpolation::setInput(MaskType::Pointer input)
 {
-    if (!input0 || !input1 || !input2 || !input3)
-        return;
-
-    d->input.push_back(input0);
-    d->input.push_back(input1);
-    d->input.push_back(input2);
-    d->input.push_back(input3);
+    if (input)
+    {
+        d->input.push_back(input);
```

Comment:
Oh, sorry, i forgot! I already answered on the ... how can we call it, hum... the "main" page of this PR: https://github.com/LoicCadour/medInria-public/pull/70, the message 25 days ago :)